modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -263,6 +264,141 @@ Marketplace URL: https://marketplace.owncloud.com/apps/contacts[Contacts]
 
 For commands for managing contacts, please see the DAV Command section in the occ core command set.
 
+[[files_Antivirus]]
+Anti-Virus
+----------
+
+Marketplace URL: https://marketplace.owncloud.com/apps/files_antivirus[Anti-Virus]
+
+Use these commands to configure the Anti-Virus app.
+Parametrisation must be done with the `occ config` command set.
+
+List the current settings
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+....
+sudo -u www-data php occ config:list files_antivirus
+....
+
+Set the setting
+~~~~~~~~~~~~~~~~
+
+To set a new value, use the command below and replace <Key> and value <Value> accordingly.
```

Comment:
I'd suggest the following so that "Key" and "Value" stand out more:

```rst
`<Key>` and value `<Value>` accordingly
```

Reply:
NP, will do including in brute force as we have the same thing there.